docs/blockchain/contracts/nft-contract/nft-actions/create.b.md

```diff
@@ -0,0 +1,359 @@
+---
+title: 'create.b'
+order: 10
+deploy: ['staging', 'mainnet']
+---
+
+# create.b
+
+Create a token factory.
+
+## Behavior
+
+Used to create a token factory for the asset_creator and will be managed by the asset_manager.
+
+The asset_manager and the asset_creator need to agree on the token factory configuration according to their specific business strategy, like resale share, etc. The agreement is done via the co-signing of the transaction.
+
+By creating the token factory together, the asset_creator agrees to all the terms, including letting the asset_manager manage the factory and its tokens.
+
+If the stat parameter is not specified, a new token factory is created in inactive state which may be changed with the setstatus action.
+
+RAM usage of a factory creation is covered by eosio.nftram.
+
+If asset_manager is other than ultra.nft.ft, the cost of a factory creation is paid in UOS to eosio.pool.
+
+## Technical Behavior
+
+Upon creation, a token factory id will be automatically assigned to the new token factory. This id is incremental.
+
+**Token factory ID**
+
+-   64-bit number
+
+-   Factory counter is stored in a singleton table, which will be automatically increased each time a token factory is created.
+
+**Factory creation**
+
+-   RAM usage of a factory creation is covered by eosio.nftram. 4GB will be gifted to eosio.nfrram to start with. The action fails If the unused RAM of eosio.nftram is less than or equal to 200MB.
+
+-   factory data is stored to factory.b table. Each factory.b entry’s pack size should be less than or equal to 1920 bytes.
+
+-   if asset_manager is other than ultra.nft.ft, The cost of a factory creation is paid to eosio.pool and it is non-refundable.
+
+    -   First, the cost in USD is (factory RAM payment size) \* (RAM price), where
+
+        -   factory RAM payment size: 2KB
+
+        -   RAM price: 0.15USD/KB
+
+    -   The cost is paid in UOS. The action gets 1 MINUTE conversion rate in USD/UOS from eosio.oracle contract. and calculates the cost by
+        (2KB \* 0.15USD/KB) / (conversion rate) = 0.3USD/(conversion rate)
+
+**Authorized minters registration**
+
+-   Authorized minters can be registered at the same time, by being specified as authorized_minters (an array of minter_authorization_info) parameter.
+
+-   Registration cost is calculated and charged to asset_manager. For the details, see authorized minters info RAM usage/cost calculation and payment/refund.
+
+Minting limit per account of a token factory
+Minting limit can also be set at the same time, by being specified as account_minting_limit parameter.
+
+**Notifications**
+
+`require_recipient` is done for `asset_manager` and `asset_creator`
+
+## Action Parameters
+
+Try to think of the action parameters as a **JSON Object** when reading this table. There will be a **JavaScript** example of the action below this table.
+
+<table>
```

Comment:
Why was the markdown table replaced with HTML table? Is it necessary to do in other documents too?